lib/helpers/defaultConfiguration.js

```diff
@@ -19,7 +19,7 @@ module.exports = {
         maxRetryCount           : 10,
         validatePublisher       : false,
         validateVersion         : false,
-        dispatchType            : 'serial',
+        dispatchType            : 'concurrent',
```

Comment:
I have it set to use `serial` as the safe option.  Most users of a queue have an expectation of adopting a behavior of FIFO.  While RabbitMQ delivers messages FIFO to the door step of the client driver, it is up to the client driver to continue that behavior in its ecosystem.  In the case of `amqplib`, it fires off the handler in FIFO ordering, but doesn't respect the settlement ordering of those handlers it fired off.  I think this is a flaw with `amqplib`.   It is why I implemented the dispatcher to help augment that behavior.  I understand that `serial` type is on an order of 10x magnitude slower than `concurrent`.  I think this is acceptable if ordering is important where a the next message can only be handled after the current one is done.

Reply:
Yeah some of this is node runtime complexity. If you effectively yield to the event loop in your handler then you won't settle events in the same order - but that's on the library user, we're still calling handles in the correct FIFO way in the library/driver. 

I see `serial` as being a library enhancement to help developers ensure FIFO, but given it's extreme performance penalty I believe it should be opt in. Maybe it's worth adding in some docs around this to make things clear?

I suspect we may be able to rework the serial handler to be more efficient, but that is it's own bunny trail.

Reply:
Ping me if you end up contributing to a better serial dispatcher.   As of right now, The speed isn't an issue when we compound the out of process call time with other things.  But it would be nice to make it go faster.